pinc/bootstrap.inc

```diff
@@ -30,6 +30,10 @@ spl_autoload_register('dp_class_autoloader');
 // Load composer autoloader
 require_once($relPath.'../vendor/autoload.php');
 
+// Include misc.inc which contains useful functions that almost 90% of
+// the pages use in some way, including our exception handlers.
+include_once($relPath.'misc.inc');
```

Comment:
There is a "DPDatabase::escape" in misc.inc. Could that cause a problem?

Reply:
Good catch. It isn't technically a problem because `set_col_str()` isn't called from the handlers so PHP won't care. It's a little unfortunate that there's a function in `misc` that relies on other DP code though. Ah look, there's `dp_setcookie()` used in the csrf_token code too.

So while technically this works I think I'll update the exception handlers to escape the HTML differently.